pkg/apis/operator/v1alpha1/knativeserving_types.go

```diff
@@ -126,11 +95,6 @@ type KnativeServingSpec struct {
 type KnativeServingStatus struct {
 	duckv1.Status `json:",inline"`
```

Comment:
I can see both eventing and serving having identical status fields, including version, conditions, and possibly `DryRunPatch` if that ever comes to fruition. I know we can account for that through the `Status` interface, but it seems cumbersome to continue replicate the fields in both the `KnativeServingStatus` and `KnativeEventingStatus` structs. Thoughts?

Reply:
We can certainly try to reduce boilerplate. In this workstream I'm more interested in collapsing business logic code vs. trying to minimize boilerplate on the typing side though. Imo, reusing business logic is the big value add here.